fw/hpack.c

```diff
@@ -702,11 +702,7 @@ tfw_huffman_decode(TfwHPack *__restrict hp, TfwHttpReq *__restrict req,
 				hp->curr -= shift;
 			}
 			else {
-				/*
-				 * @shift for short tables must be greater
-				 * than zero.
-				 */
-				WARN_ON_ONCE(1);
+				T_ERR("Decoding error. Short table shift less than zero.");
```

Comment:
Usually we use `T_ERR` when happened something critical, like OOM. Please use `T_WARN` instead.